classes/Topic.php

```diff
@@ -0,0 +1,339 @@
+<?php
+/**
+ * Topic
+ *
+ * @package TheyWorkForYou
+ */
+
+namespace MySociety\TheyWorkForYou;
+
+class Topic {
+
+    /**
+     * DB handle
+     */
+    private $db;
+
+    private $raw;
+    private $id;
+    private $title;
+    private $slug;
+    private $description;
+    private $image;
+    private $search_string;
+    private $front_page;
+    private $actions;
+
+    /**
+     * Constructor
+     *
+     */
+
+    public function __construct($data = NULL)
+    {
+        $this->db = new \ParlDB;
+
+        if (is_null($data)) {
+          return;
+        }
+
+        $this->raw = $data;
+        if (isset($data['id'])) {
+            $this->id = $data['id'];
+        }
+        $this->title = $data['title'];
+        $this->slug = $data['slug'];
+        $this->description = $data['description'];
+        $this->image = $data['image'];
+        if (isset($data['search_string'])) {
+            $this->search_string = $data['search_string'];
+        }
+        if (isset($data['front_page'])) {
+            $this->front_page = $data['front_page'];
+        }
+
+        $this->raw['actions'] = array();
+    }
+
+
+    function data() {
+        return $this->raw;
+    }
+
+    function title() {
+        return $this->title;
+    }
+
+    function set_title($title) {
+        $this->title = $title;
+    }
+
+    function slug() {
+        return $this->slug;
+    }
+
+    function set_slug($slug) {
```

Comment:
Probably not worth it here, and I haven't seen what other classes do, but do/could we consider having `slug` be a getter and a setter?